### PR TITLE
First modification

### PR DESCRIPTION
**Pull request by CelestialDemon**
Hello adrian, just added a new print, maybe you want to update it!!!

### CODE REVIEW

**Comment by adrytheboss** [comment]
Hello adrian, please add another exaple and later squash both commits into one.

File: hello.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
 print('hello world')
 print('Modification done from the GitHub Web')
```
Comment (adrytheboss) [inline hello.py:2]: This line is good, just add a couple more

**Comment by adrytheboss** [comment]
Great adrian, I love this! You did a fenomenal job

File: hello.py (hunk 1 above)
Comment (CelestialDemon) [reply hello.py:2]: Done

**Pull request merged**